src/utils/messages.ts

```diff
@@ -2,6 +2,12 @@ export default {
   VALIDATION_FAIL: {
     title: 'Validation failed',
     message: 'Please fix invalid fields before continuing',
+    option: 'Yes',
```

Comment:
```suggestion
    option: 'OK',
```
The norm for these kind of buttons is "OK", I think.